ansible_collections/arista/avd/roles/eos_designs/python_modules/interface_descriptions/load_interface_descriptions.py

```diff
@@ -0,0 +1,35 @@
+import importlib
+
+from ansible_collections.arista.avd.plugins.plugin_utils.utils import AristaAvdError, get
+
+from .avdinterfacedescriptions import AvdInterfaceDescriptions
+
+DEFAULT_AVD_INTERFACE_DESCRIPTIONS_PYTHON_MODULE = "ansible_collections.arista.avd.roles.eos_designs.python_modules.interface_descriptions"
+DEFAULT_AVD_INTERFACE_DESCRIPTIONS_PYTHON_CLASS_NAME = "AvdInterfaceDescriptions"
+
+
+def load_interfacedescriptions(hostvars, templar) -> AvdInterfaceDescriptions:
```

Comment:
probably can write some common loader for descriptions and ip_address in some generic utils file so that it can be reused consistently

Looks like it would need as input:
* variable path `switch.<x>` (and the loader can append python_module and python_class_name)
* some DEFAULT_PYTHON_MODULE and DEFAULT_PYTHON_CLASS_NAME which can be give as argument
* the subclass type


the variables could be located in the __init__.py file and the load_python_module class made available from there.